src/python/grpcio_tests/tests/unit/_rpc_test.py

```diff
@@ -186,12 +218,17 @@ def setUp(self):
         self._handler = _Handler(self._control)
         self._server_pool = logging_pool.pool(test_constants.THREAD_CONCURRENCY)
 
-        self._server = grpc.server(self._server_pool)
+        self._server_interceptor = _ServerInterceptor()
```

Comment:
I'd prefer to see these tests moved into their own _interceptor_test.py file, by changing the setup logic in rpc_test.py we have lowered test coverage on the standard ```grpc.insecure_channel``` and ```grpc.server```.